### ISSUE TITLE
Get-ODAuthentication : A parameter cannot be found that matches parameter name 'ResourceId'.

### ISSUE THREAD

**Issue by Mahmoud87** [MarcelMeurer/PowerShellGallery-OneDrive#19]
Installed the module and went through the app registration but when I try to get the token I get this error.

PS C:\WINDOWS\system32> $Auth=Get-ODAuthentication -ClientId "a2xxcd-xx-4cd3-b234b-22acccsssxxx32" -AppKey "xxxxxx00032Q_l0~9xxxxxppooyumiz" -RedirectURI "http://localhost/1Drive4Business" -ResourceId "https://companyname.sharepoint.com/"
```
Get-ODAuthentication : A parameter cannot be found that matches parameter name 'ResourceId'.
At line:1 char:122
+ ... b32" -AppKey "xxx~i1y32Q_l0~9-Nwww12xx_7yumiz" -ResourceId "https: ...
+                                                       ~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-ODAuthentication], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Get-ODAuthentication
```

Am I using the redirect URL correctly? Is it mandatory to have this parameter? How to determine the correct  Resource ID? 

Thanks

**Comment by RedIkki** [comment]
Yes, it is mandatory as it will be used to actually connect to OneDrive. 

I've stumbled across this issue sometimes now but the strange thing is that it's totally random apparently. Got that in a machine and after some tries, it worked. But in other machines it doesn't work at all.
